src/app/pages/send/send-crypto-asset-form/stacks-sip10/stacks-sip10-fungible-token-send-form.tsx

```diff
@@ -120,8 +130,16 @@ export function StacksSip10FungibleTokenSendForm({
       validateOnMount={false}
       validationSchema={validationSchema}
     >
-      <Form>
-        <AmountField symbol={symbol.toUpperCase()} rightInputOverlay={<></>} />
+      <Form style={{ width: '100%' }}>
```

Comment:
`form` is a block level element, so should be width 100% by default

Reply:
I actually had to add this bc during typing in the amount field the form would resize, any other thoughts then on why that would be?

Reply:
When it is removed...
![Screen Shot 2023-01-25 at 10 26 57 AM](https://user-images.githubusercontent.com/6493321/214619872-72e3559b-4067-4b48-8c03-1102f52ad3e5.png)